app/views/teams/show.html.erb

```diff
@@ -35,17 +35,17 @@
       <thead>
           <tr>
             <th><%= t '.select', :default => t("helpers.select.prompt") %></th>
-            <th><%= model_class.human_attribute_name(:id) %></th>
+            <th>#</th>
```

Comment:
Consider using a translation string here as well be stay consistent. You would have to adjust a test case as well.